modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/taglib/_search_container.scss

```diff
@@ -88,6 +88,11 @@
 .lfr-description-column, .lfr-role-column {
 	max-width: 375px;
 	min-width: 375px;
+
+	// &.lfr-search-iterator-fixed-table-data {
+	// 	max-width: 225px;
+	// 	min-width: 225px;
+	// }
```

Comment:
Was this intended to remain commented, or should we remove it?

Reply:
Oh man, good catch, yes it should be removed